ocs_ci/deployment/ocp.py

```diff
@@ -44,9 +44,13 @@ def download_pull_secret():
 
 
 class OCPDeployment:
-    def __init__(self):
+    def __init__(self, **kwargs):
         """
         Constructor for OCPDeployment class
+
+        Args:
+            **kwargs: keyword arguments;
+            skip_download_installer for skipping download OCP installer regardless of the configuration
```

Comment:
This can be formatted (at least in docstring) as a proper parameter

Reply:
this is a part of kwargs, another set pair. pls show the example for this case I will adjust